lua/obsidian/command.lua

```diff
@@ -144,12 +144,50 @@ command.backlinks = function(client, _)
   vim.cmd "lop"
 end
 
+---Search notes.
+---
+---@param client obsidian.Client
+---@param data table
+command.search = function(client, data)
+  local base_cmd = vim.tbl_flatten { util.SEARCH_CMD, { "--column", "--line-number", "--no-heading" } }
+
+  if vim.fn.exists "*fzf#vim#grep" then
```

Comment:
I guess here we have to also check for `fzf#vim#with_preview`

Reply:
Done: https://github.com/epwalsh/obsidian.nvim/pull/27/commits/36cc6c88ddc23031937b772d5077091cd563a247

Reply:
Unfortunately the same error prevails! :( It is to late today, I will check it out tomorrow.

Reply:
Hmm. Weird. Okay well I changed it so that the check for telescope comes first. That should fix it for you.